docs/src/main/paradox/s3.md

```diff
@@ -93,9 +101,15 @@ Copy an S3 object from source bucket to target bucket using multi part copy uplo
 Scala
 : @@snip ($alpakka$/s3/src/test/scala/akka/stream/alpakka/s3/scaladsl/S3SinkSpec.scala) { #multipart-copy }
 
+Scala
+: @@snip ($alpakka$/s3/src/test/scala/akka/stream/alpakka/s3/scaladsl/S3SinkSpec.scala) { #multipart-copy-with-source-version }
+
```

Comment:
You should make this two snippet sections, and add some text explaining what it does.